tests/sanity/utils.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+
+import subprocess
+
+
+def run_in_docker(image, check_exit_code, *command):
+    return subprocess.run(
+        [
+            "docker",
+            "run",
+            "--rm",
+            "-i",
```

Comment:
I guess we can drop the ``-i`` flag, we don't really intend to use stdin.

Reply:
Done